app/services/jobseekers/job_applications/prefill_job_application_from_previous_application.rb

```diff
@@ -28,13 +28,11 @@ def copy_personal_info
       new_job_application.assign_attributes(recent_job_application.slice(*(attributes - [:baptism_certificate])))
 
       if recent_job_application.baptism_certificate.present?
-        recent_job_application.baptism_certificate.blob.open do |tempfile|
-          new_job_application.baptism_certificate.attach({
-            io: tempfile,
```

Comment:
I think this line is what was causing the existing baptism certificate bug. We were assigning tempfile here, but by the time we went to save the job application on [line 15](https://github.com/DFE-Digital/teaching-vacancies/blob/4dbb2c2ef0d58e2ab71522bb04748076b0ab8087/app/services/jobseekers/job_applications/prefill_job_application_from_previous_application.rb#L15) tempfile is out of scope, the file no longer exists so we get [this error](https://teaching-vacancies.sentry.io/issues/7084973182/?environment=review&query=is%3Aunresolved&referrer=issue-stream).